NashTeamAlpha/CustomerIncidentPortal
# Db testing intro

## Pull Request Description

1. created two tests to create a new employee.
.1 then created code to make these tests pass. 

## Steps to Test

Include the applicable information from the list below here

1. git fetch
2. git checkout db-testingIntro
3. build the solution
4. run the tests. They should both pass. 

## Link to Feature Ticket

Provide the issue number that this pull request fixes
Example: #35
